src/index.ts

```diff
@@ -107,6 +114,13 @@ module.exports = class ServerlessPlugin {
       this.serverless.cli.log("Adding Datadog Lambda Extension Layer to functions");
       this.debugLogHandlers(handlers);
       applyExtensionLayer(this.serverless.service, handlers, allLayers);
+      handlers.forEach((functionInfo) => {
+        if (functionInfo.type === RuntimeType.DOTNET) {
+          this.serverless.cli.log("Adding Dotnet Tracing Layer to functions");
```

Comment:
Dotnet -> .NET

The official name is .NET, we just use dotnet in contexts where a dot isn't allowed.